portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,30 +29,48 @@
 /** Servlet that returns some example content. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private List<String> comments = new ArrayList<>(
-    Arrays.asList("Nice!",
-                  "Very interesting.",
-                  "I would like to know more about your favourite TV shows!"));
+  private final Deque<Comment> comments = new LinkedList<>();
 
   /**
-   * Converts a List instance into a JSON string.
+   * Converts a Deque instance into a JSON string.
    */
-  public static String convertListToJson(List<String> list) {
-    StringBuilder jsonBuilder = new StringBuilder();
-    jsonBuilder.append("[");
-    for (int i = 0; i < list.size(); i++) {
-      jsonBuilder.append("\"" + list.get(i) + "\"");
-      if (i < list.size() - 1) {
-        jsonBuilder.append(", ");
-      }
-    }
-    jsonBuilder.append("]");
-    return jsonBuilder.toString();
+  public static String convertDequeToJson(Deque<Comment> deque) {
```

Comment:
when naming variables, don't include their type in the name. The type is already defined anyway so no need to repeat yourself. Instead make the name about the semantics, i.e. what is being stored.

Reply:
Ok. Because before that, it was a deque of strings and I wanted to keep it general, not specific to comments, so I called it deque. But now using the Comment class, I can use a more specific name.